sky/engine/core/compositing/SceneBuilder.idl

```diff
@@ -14,6 +14,7 @@
   void pop();
 
   void addPicture(Offset offset, Picture picture, Rect bounds);
+  void addStatistics(unsigned long enabledOptions, Rect bounds);
```

Comment:
How does the caller know what bounds to use?  I guess you pass the bounds of the whole screen and hope the stats fit?